### DISCUSSION TITLE
How to use the tailwindcss plugin?

### DISCUSSION THREAD

**Discussion by adb-sh** [lumeland/lume#389] [Q&A]
Hey :) i followed the steps in the docs but i didn't get it working :/ (no styles are added)
Is there a demo project, that shows how to install the tailwindcss plugin correctly?
Tested with .njx and .tsx template files.

**Comment by oscarotero** [comment]
Tailwind requires to add the `@tailwind` at rules at some point in your CSS code. For example:
https://github.com/lumeland/lume/blob/master/tests/assets/tailwindcss/styles.css

I'll update the docs to make it more clear

**Reply by adb-sh** [reply]
Ahh, totaly forgot about this. Thank you.
